capplets/appearance/appearance-ui.c

```diff
@@ -20,7 +20,9 @@
  */
 
 #include "appearance.h"
-#include "stdio.h"
+#include "appearance-ui.h"
+
+#include <stdio.h>
```

Comment:
I think we can remove this header file, it doesn't work

Reply:
```
$ git grep ui_init
capplets/appearance/appearance-main.c:  ui_init (data);
capplets/appearance/appearance-ui.c:ui_init (AppearanceData *data)
capplets/appearance/appearance-ui.h:void ui_init (AppearanceData *data);
```

Reply:
You may have misunderstood me. I mean ```stdio.h``` here

Reply:
You don't need to remove `#include <stdio.h>` as the `NULL` macro is defined in the `stdio.h` file.
However, we need to check the includes as they are missing, such as `#include <glib.h>`, `#include <gtk/gtk.h>`, `#include <gio/gio.h>`.
I will look for which includes are missing with the eclipse analyzer.